test/helpers/PlayerInteractionWrapper.js

```diff
@@ -142,7 +142,14 @@ class PlayerInteractionWrapper {
     }
 
     get deck() {
-        return this.player.deck;
+        return this.player.drawDeck;
+    }
+
+    setDeck(newContents = []) {
+        newContents.reverse().forEach((name) => {
```

Comment:
Do we need to empty the deck before doing this, or is the intent that we're just moving a subset of cards to the top?